### PR TITLE
PSR-1019 postman replaced with bruno and login mechanism changed

### PR DESCRIPTION
**Pull request by cyeniceri**

### CODE REVIEW

File: test/resources/PSR with Login/environments/LocalHost - PSR - with LOGIN.bru [hunk 1]
```diff
@@ -0,0 +1,5 @@
+vars {
+  pension-scheme-return: http://localhost:10700/pension-scheme-return
+  auth: http://localhost:8585
+  bearer_token: BXQ3/Treo4kQCZvVcCqKPlJ4wf4lIOaO27ua3kjMs3S6iFOAsYhjaQAEtHMEd7377fZnbGSgp5WiM4DlZst2/pxfAbw+lYTjsnYfjyofDN1OhVf06Xj5KByFuciwhPWi05Xz1oCm3LdYKDST9N0epHaDkKrec9hPESB1GeXRgB79KwIkeIPK/mMlBESjue4V
```
Comment (kberec) [inline test/resources/PSR with Login/environments/LocalHost - PSR - with LOGIN.bru:4]: Is it right to have this version controlled?

File: test/resources/PSR with Login/PSR-Login.bru [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+post {
+  url: {{auth}}/government-gateway/session/login
+  body: json
+  auth: none
+}
+
+headers {
+  Authorization: Bearer BXQ3/Treo4kQCZvVcCqKPn7sOksB9wQwDo28f+J2ns1djM1UYrnoRbtOcgt47BuscniXI8OSDhXn90ppX+CnLO9Ek0vYb0NJVplmXggKqS8OOhJ7w0MM2y5rvhyy6F5CI0LPqneRFViZn51rt5XJIkv/Ajp8u4ZYaxEDyMpHh0z9KwIkeIPK/mMlBESjue4V
```
Comment (kberec) [inline test/resources/PSR with Login/PSR-Login.bru:14]: Is it right to have this version controlled?

File: test/resources/PSR with Login/Overview - Invalid date range.bru [hunk 3]
```diff
@@ -0,0 +1,26 @@
+meta {
```
Comment (kberec) [inline test/resources/PSR with Login/Overview - Invalid date range.bru:1]: We'd need a Readme for bruno please.

File: test/resources/PSR with Login/Overview - Invalid date range.bru [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  name: Overview - Invalid date range
```
Comment (kberec) [inline test/resources/PSR with Login/Overview - Invalid date range.bru:2]: I'm getting the following response on login:
```
{
  "gatewayToken": "unknown-token"
}
```
It still works, but is the above expected?

File: test/resources/PSR with Login/environments/LocalHost - PSR - with LOGIN.bru (hunk 1 above)
Comment (cyeniceri) [reply test/resources/PSR with Login/environments/LocalHost - PSR - with LOGIN.bru:4]: removed

File: test/resources/PSR with Login/PSR-Login.bru (hunk 2 above)
Comment (cyeniceri) [reply test/resources/PSR with Login/PSR-Login.bru:14]: removed

File: test/resources/PSR with Login/Overview - Invalid date range.bru (hunk 3 above)
Comment (cyeniceri) [reply test/resources/PSR with Login/Overview - Invalid date range.bru:1]: added

**Review by kberec — APPROVED** [review]

**Pull request merged**